aya-log/src/lib.rs

```diff
@@ -90,7 +90,7 @@ impl BpfLogger {
         logger: T,
     ) -> Result<BpfLogger, Error> {
         let logger = Arc::new(logger);
-        let mut logs: AsyncPerfEventArray<_> = bpf.map_mut("AYA_LOGS")?.try_into()?;
+        let mut logs: AsyncPerfEventArray<_> = bpf.take_map("AYA_LOGS").unwrap().try_into()?;
```

Comment:
this `?` -> `.unwrap()` means that bpf programs that don't log will result in a panic on the userspace side.

Sent #476 to fix this.